ignite/pkg/cosmosanalysis/app/testdata/modules/api_routes/app.go

```diff
@@ -10,6 +10,8 @@ import (
 	authtx "github.com/cosmos/cosmos-sdk/x/auth/tx"
 	"github.com/cosmos/cosmos-sdk/x/bank"
 	"github.com/cosmos/cosmos-sdk/x/gov"
+	govclient "github.com/cosmos/cosmos-sdk/x/gov/client"
+	paramsclient "github.com/cosmos/cosmos-sdk/x/params/client"
```

Comment:
Why were the `x/params` module imports added to the test app files? If possible it might be a good idea to remove them if they are not required given that the module is [deprecated](https://github.com/cosmos/cosmos-sdk/blob/release/v0.47.x/UPGRADING.md#xparams).